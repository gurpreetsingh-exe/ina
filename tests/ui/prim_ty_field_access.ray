// COMPILE_FAIL

fn main() {
    let a = 20;
    let b = a.a;
}

