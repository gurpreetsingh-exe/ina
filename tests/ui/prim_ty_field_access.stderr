error: invalid field access on primitive type
   ╭─[tests/ui/prim_ty_field_access.ina:5:13]
   │
 5 │     let b = a.a;
   │             ┬
   │             ╰─ `integer` has no field `a`
   ∙
───╯
